packages/front-end/components/Layout/Layout.tsx

```diff
@@ -27,9 +27,38 @@ const navlinks: SidebarLinkProps[] = [
     name: "Features",
     href: "/features",
     Icon: BsFlag,
-    path: /^features/,
-    beta: false,
-    className: styles.first,
+    path: /^(features|attributes|namespaces|environments|saved-groups)/,
+    autoClose: true,
+    subLinks: [
+      {
+        name: "All Features",
+        href: "/features",
+        path: /^features/,
+      },
+      {
+        name: "Attributes",
+        href: "/attributes",
+        path: /^attributes/,
+        permissions: ["organizationSettings"],
+      },
+      {
+        name: "Namespaces",
+        href: "/namespaces",
+        path: /^namespaces/,
+        permissions: ["organizationSettings"],
+      },
+      {
+        name: "Environments",
+        href: "/environments",
+        path: /^environments/,
```

Comment:
There are a few links in our app to the old environments URL.  Make sure to update those.  Search for `/settings/environments` to find them.